src/authentication/privateRoutes.tsx

```diff
@@ -0,0 +1,11 @@
+import { useAuth } from "./useAuth.ts";
+import { Navigate, Outlet } from "react-router-dom";
+import * as React from "react";
+
+const PrivateRoutes: React.FC = () => {
```

Comment:
is this type necessary?

Reply:
no, it was a "leftover" of some problem I had encountered. removed!